### ISSUE TITLE
API Pay Model

### ISSUE THREAD

**Issue by itmilos** [itmilos/itmilos#4]
All data used for https://github.com/itmilos/uplatnica/issues/7 should be available via open API

**Comment by misaang** [comment]
All data sources should be from the trusted sources, government sites and all other official public institutions

**Comment by itmilos** [comment]
Indeed, ref. https://www.nbs.rs/documents/ostalo/SifrePlacanjaDinarske.xml

**Comment by itmilos** [comment]
Related to for https://github.com/Code-for-Serbia/uplatnica/issues/7 should be available via open API
